lib/app/upload.ex

```diff
@@ -4,8 +4,7 @@ defmodule App.Upload do
   """
   import SweetXml
 
-  @region Application.compile_env(:ex_aws, :region)
-  @compressed_baseurl "https://s3.#{@region}.amazonaws.com/imgup-compressed/"
+  @compressed_baseurl "https://s3.eu-west-3.amazonaws.com/imgup-compressed/"
```

Comment:
The `original` and `compressed` bucket should be an env variable, IMO. They are used across the application and it would be easier to have them as env variable to change as needed much more easier.

Reply:
100%. We can update it to _not_ be hard-coded once we've confirmed that this is working. 🤞

Reply:
I briefly had it as an Env Var but it was causing issues so I hard-coded it _temporarily_.

Reply:
Rog 🆗 
Created the issue in https://github.com/dwyl/imgup/issues/80.